kolena/workflow/test_run.py

```diff
@@ -242,8 +242,7 @@ def upload_inferences(self, inferences: List[Tuple[TestSample, Inference]]) -> N
         df_serializable = df_validated.as_serializable()
 
         init_response = init_upload()
-        upload_data_frame_chunk(df_serializable, init_response.uuid)
-
+        upload_data_frame_in_smart_chunks(df_serializable, init_response.uuid)
```

Comment:
I think the `upload_data_frame_chunk` in these places should have used `upload_data_frame` in the first place. For this PR, we should be updating occurences of `upload_data_frame` with the "new method".

I would suggest renaming current `upload_data_frame` to internal method, and `upload_data_frame_in_smart_chunks` to `upload_data_frame`. (note: signature changed tho).

cc @gordonhart for any concerns.

Reply:
Will update